spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/services/SCConfigService.java

```diff
@@ -245,10 +204,14 @@ public void addConfig(File configFile) {
     public void start() {
         Log.d(LOG_TAG, "Starting SCConfig Service.  Loading all configs");
         loadConfigs();
-        if (API_URL != null) {
-            loadRemoteConfig();
-            registerDevice();
-        }
+        SCBackendService.initialized.subscribe(new Action1<Integer>() {
```

Comment:
This needs to work like the ios sdk, where the backend service is started, and after authenticating, the remote config is fetched from the mqtt topic